modules/ROOT/pages/administration/access-control/manage-privileges.adoc

```diff
@@ -195,10 +195,149 @@ The following image shows the hierarchy between different graph privileges:
 
+[[access-control-list-supported-privileges]]
+== Listing supported privileges
+
+Supported privileges can be displayed using the `SHOW SUPPORTED PRIVILEGES` command.
+This lists the privileges that are possible to grant or deny on this server, together with the structure of the privilege.
+
+.Show supported privileges command syntax
+[cols="<15s,<85"]
+|===
+
+| Command
+m| +SHOW SUPPORTED PRIVILEGES+
+
+| Syntax
+a|
+[source, syntax, role="noheader", indent=0]
+----
+SHOW SUPPORTED PRIVILEGE[S]
+  [YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+  [WHERE expression]
+  [RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+----
+| Description
+| List all privileges supported by the server.
+
+|===
+
+When using the `RETURN` clause, the `YIELD` clause is mandatory and must not be omitted.
+
+Results will include multiple columns describing the privileges:
+
+[options="header", width="100%", cols="4m,6a,2m"]
+|===
+| Column | Description | Type
+
+| action
+| The privilege action
+| STRING
+
+| qualifier
+| Qualifier to further limit the target of the privilege, null if not applicable
+| STRING
+
+| target
+| Target of the privilege: `dbms`, `database` or `graph`
+| STRING
+
+| scope
+| List of possible scopes for the privilege, null if not applicable
+| LIST OF STRING
+
+| description
+| A short description of the privilege
+| STRING
+
+|===
+
+[[access-control-list-supported-privileges-example]]
+=== Examples for listing supported privileges
+
+[source, cypher, role=noplay]
+----
+SHOW SUPPORTED PRIVILEGES YIELD * ORDER BY action DESC LIMIT 10 RETURN action, qualifier, target, scope, description
+----
+
+Lists 10 supported privileges:
+
+.Result
+[options="header,footer", width="100%", cols="m,m,m,m,m"]
+|===
+|action
+|description
+|target
+|scope
+|qualifier
+
+| "show settings"
+| "setting"
+| "dbms"
+| NULL
+| "enables the specified roles to query given configuration settings"
+
+| "show transactions"
+| "username"
+| "database"
+| NULL
+| "allows listing transactions and queries for the specified users on the specified database"
+
+| "show user"
+| NULL
+| "dbms"
+| NULL
+| "enables the specified roles to list users"
+
+| "start"
+| NULL
+| "database"
+| NULL
+| "allows the specified database to be started"
+
+| "stop"
+| NULL
+| "database"
+| NULL
+| "allows the specified database to be stopped"
+
+| "terminate transactions"
+| "username"
+| "database"
+| NULL
+| "allows ending transactions and queries for the specified users on the specified database"
+
+| "transaction management"
+| "username"
+| "database"
+| NULL
+| "allows listing and ending transactions and queries for the specified users on the specified database"
+
+| "traverse"
+| NULL
+| "graph"
+| ["elements", "nodes", "relationships"]
+| "enables the specified entities to be found"
+
+| "user management"
+| NULL
+| "dbms"
+| NULL
+| "enables the specified roles to create, delete, modify, and list users"
+
+| "write"
```

Comment:
If we order descending, shouldn't these be in the opposite order? so write is first and show settings last?

Reply:
that is copied from the output of cypher-shell, so I didn't look closely at it, but it seems wrong

Reply:
Might have missed adding the new show command to `normalizeWithAndReturnClauses`, not sure if it will help but it seems to be missing from there anyway 🤔